### ISSUE TITLE
bug(testbot): Request retesting of multiple tests seems to be broken

### ISSUE THREAD

**Issue by ermshiperete** [keymanapp/status.keyman.com#264]
It seems it's no longer possible to request retesting of multiple tests in one comment.

I tried all kinds of combinations, e.g. `@keymanapp-test-bot retest SUITE_OSK GROUP_JAMMY_X11 all, SUITE_OSK GROUP_JAMMY_WAYLAND all` and it simply gets ignored. It seems it works in most cases, but in https://github.com/keymanapp/keyman/issues/6528 it doesn't - maybe because of the number of tests?

**Comment by ermshiperete** [comment]
I re-created that specific issue in the playground, so that can be used to test and play around (https://github.com/keymanapp/keymanapp-test-bot-playground/issues/12).

**Comment by ermshiperete** [comment]
and with an additional comment it works in the original issue now as well. Strange...

**Comment by mcdurdin** [comment]
```
2022-06-09T20:58:00.380160387Z statusCode for api.github.com/graphql: 502
2022-06-09T20:58:00.399403081Z TypeError: Cannot read properties of null (reading 'start')
2022-06-09T20:58:00.399434581Z     at /home/site/wwwroot/server/services/github/github-status.ts:27:30
2022-06-09T20:58:00.399441281Z     at runMicrotasks (<anonymous>)
2022-06-09T20:58:00.399446581Z     at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

and 

```
2022-06-09T21:01:41.468152842Z Error: socket hang up
2022-06-09T21:01:41.468203442Z     at connResetException (node:internal/errors:691:14)
2022-06-09T21:01:41.468210842Z     at TLSSocket.socketOnEnd (node:_http_client:471:23)
2022-06-09T21:01:41.468214942Z     at TLSSocket.emit (node:events:402:35)
2022-06-09T21:01:41.468218642Z     at TLSSocket.emit (node:domain:475:12)
2022-06-09T21:01:41.468222442Z     at endReadableNT (node:internal/streams/readable:1343:12)
2022-06-09T21:01:41.468226342Z     at processTicksAndRejections (node:internal/process/task_queues:83:21) {
2022-06-09T21:01:41.468230342Z   code: 'ECONNRESET'
2022-06-09T21:01:41.468234042Z }
```

**Comment by mcdurdin** [comment]
Restarted the server and that restored things. So there is a robustness issue when we get errors connecting to GitHub (and probably other services also). My thought is that when we get these errors, we should reflect that in the interface with the refresh button turning into a warning triangle. But that's back burner for now while I deal with other crises ;-)

**Comment by mcdurdin** [comment]
Also

```
2022-06-09T22:36:13.233043809Z Error: getaddrinfo ENOTFOUND linux.lsdev.sil.org
2022-06-09T22:36:13.233064909Z     at GetAddrInfoReqWrap.onlookup [as oncomplete] (node:dns:71:26) {
2022-06-09T22:36:13.233120609Z   errno: -3008,
2022-06-09T22:36:13.233128009Z   code: 'ENOTFOUND',
2022-06-09T22:36:13.233132209Z   syscall: 'getaddrinfo',
2022-06-09T22:36:13.233136509Z   hostname: 'linux.lsdev.sil.org'
2022-06-09T22:36:13.233140809Z }
```

**Comment by mcdurdin** [comment]
This should basically be resolved at this point -- robustness added in #266.
